resources/lib/utils.py

```diff
@@ -81,7 +82,12 @@ def get_json_from_response(r: Response) -> Optional[Dict]:
 
     # no content - possibly POST/DELETE request?
     if not r or not r.text:
-        return None
+        try:
+            r.raise_for_status()
```

Comment:
That's probably a good idea. I added this for the `add_to_queue` where there is no response from the api, but there might be other circumstances where we get that and it's caused by an error.